syncthing/STApplication.m

```diff
@@ -72,6 +72,16 @@ - (void)applicationLoadConfiguration {
     NSUserDefaults *defaults = [NSUserDefaults standardUserDefaults];
 
     _executable = self.arguments = [defaults stringForKey:@"Executable"];
+    if (_executable) {
+        NSFileManager *fileManager = [NSFileManager defaultManager];
+        if (![fileManager fileExistsAtPath:_executable]) {
+            NSLog(@"Resetting Syncthing daemon executable path because it doesn't exist: (%@)", _executable);
+            _executable = nil;
+        } else if (![fileManager isExecutableFileAtPath:_executable]) {
+            NSLog(@"Resetting Syncthing daemon executable path because it's non-executable: (%@)", _executable);
+            _executable = nil;
+        }
+    }
```

Comment:
Codestyle: Indent level compared with line 85 is off, not needed to change the indent level. Maybe we should auto-codeformat in the future (note to self).